src/current/v23.2/live-migration-service.md

```diff
@@ -0,0 +1,598 @@
+
+~~~ yaml
+lms:
+  shadowMode: none
+~~~
+
+`lms.shadowMode` specifies the shadowing behavior used by the LMS. This should depend on your specific migration requirements. For details, see [Shadowing modes](#shadowing-modes).
+
+#### LMS instances
+
+~~~ yaml
+lms:
+  replicaCount: 3
+~~~
+
+`lms.replicaCount` determines the number of LMS instances created as `lms-molt-lms` pods on the Kubernetes cluster, across which application traffic is distributed. This defaults to `3`.
+
+#### Source and target connections
+
+The connections to the source database and CockroachDB are defined with the configuration keys:
+
+- `INIT_SOURCE`: Connection string for the source database.
+- `INIT_TARGET`: Connection string for a CockroachDB database.
+
+Include client authentication details in the connection strings. For more details, see [Configure source and target certificates](#configure-source-and-target-certificates).
+
+{{site.data.alerts.callout_success}}
+For details about writing a CockroachDB connection string, see [Connect using a URL]({% link {{ page.version.version }}/connection-parameters.md %}#connect-using-a-url).
+{{site.data.alerts.end}}
+
+You should specify the config keys in an external Kubernetes secret and inside a JSON object. The JSON **must** be named `config.json`.
+
+For example:
+
+~~~ json
+config.json: |
+  {
+    "INIT_SOURCE": "mysql://{username}:{password}@{protocol}({host}:{port})/{database}?sslmode=verify-full?sslrootcert=path/to/mysql.ca&sslcert=path/to/mysql.crt&sslkey=path/to/mysql.key",
+    "INIT_TARGET": "postgresql://{username}:{password}@{host}:{port}/{database}?sslmode=verify-full?sslrootcert=path/to/ca.crt&sslcert=path/to/client.username.crt&sslkey=path/to/client.username.key"
+  }
+~~~
+
+For more details on storing the LMS configuration in an external secret, see [Manage external secret](#manage-external-secret).
+
+#### Certificates
+
+~~~ yaml
+lms:
+  sslVolumes: {}
+  sslVolumeMounts: {}
+...
+orchestrator:
+  sslVolumes: {}
+  sslVolumeMounts: {}
+~~~
+
+`sslVolumes` and `sslVolumeMounts` specify [volumes](https://kubernetes.io/docs/concepts/storage/volumes/#secret) and mount paths that contain server-side certificates for the LMS instances and orchestrator. For security recommendations, see [Security](#security).
+
+<a name="cert-var"></a>
+
+When you specify a volume in `sslVolumes` and `sslVolumeMounts`, you must also [set its corresponding config key](#manage-external-secret) to the path specified in `sslVolumeMounts.mountPath`.
+
+|      Key      |                        Description                         |
+|---------------|------------------------------------------------------------|
+| `SSL_CA`      | Mount path to the SSL CA certificate for the LMS.          |
+| `SSL_CERT`    | Mount path to the SSL certificate for the LMS.             |
+| `SSL_KEY`     | Mount path to the SSL key for the LMS.                     |
+| `CA_TLS_CERT` | Mount path to the TLS CA certificate for the orchestrator. |
```

Comment:
We may want to specify that this is *optional* and mainly used if you self-sign your certs. Otherwise, you can use a root CA like Let's Encrypt or AWS or etc.

Reply:
Does this also go for `SSL_CA`?